packages/devtools_app/lib/src/screens/memory/panes/allocation_tracing/allocation_profile_class_table.dart

```diff
@@ -39,9 +38,9 @@ class _TraceCheckBoxColumn extends ColumnData<TracedClass>
     VoidCallback? onPressed,
   }) {
     final controller =
-        Provider.of<MemoryController>(context).allocationTracingController;
+        Provider.of<AllocationProfileTracingViewController>(context);
```

Comment:
There may be a performance impact for pulling this from the tree on each build. We may want to consider passing the controller in as a parameter to _TraceCheckBoxColumn for this reason.